python_frank_energie/__init__.py

```
@@ -1,8 +1,15 @@
 """Frank Energie API library."""
+# python_frank_energie/__init__.py
 from .frank_energie import FrankEnergie
 from .models import Price, PriceData
+from .authentication import Authentication
+from .frank_energie import FrankEnergie
```

Ruff

Redefinition of unused `FrankEnergie` from line 3

Remove definition: `FrankEnergie`

(F811)